### PR TITLE
update overview of Mautic section TEDU-99

### PR DESCRIPTION
**Pull request by favour-chibueze**
This PR is for updates of "Welcome to the Mautic" section

<a href="https://gitpod.io/#https://github.com/mautic/user-documentation/pull/85"></a>

### CODE REVIEW

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for the work on this @favour-chibueze - I have made a bunch of recommendations for you to take a look at. Pinging @Robmobius for your thoughts on Knowledgebase or Knowledge base!

File: docs/index.rst [hunk 1]
```diff
@@ -3,7 +3,18 @@ Welcome to the Mautic documentation
...
+To get started with Mautic, follow the :doc:`Getting started guide</getting_started/how_to_install_mautic>`.
+For next steps, you can learn :doc:`how to update Mautic</getting_started/how_to_update_mautic>`.
+
+There are different types of documentation available to help you navigate your way through Mautic.
+
+:xref:`User Guide` - Describes key concepts of Mautic and provides instructions for using and contributing to Mautic.
+
+:xref:`Developer Guide` - Describes details about the Mautic API, Webhooks, Themes and Plugin development.
+
+:xref:`Knowledge Base` - Provides a database of tutorials, FAQs, and how-to articles.
```
[the remark this answers is not in the document]
Comment (RCheesley) [reply docs/index.rst:15]: @Robmobius here is the relevant part about the KB.

**Review by RCheesley — APPROVED** [review]
Good to go from my side! 🚀

**Comment by fakela** [comment]
Looks good to me 👍

**Review by fakela — APPROVED** [review]

**Pull request merged**